systemtest/src/test/java/io/strimzi/systemtest/operators/RecoveryST.java

```diff
@@ -47,6 +53,12 @@
 import static io.strimzi.systemtest.utils.kafkaUtils.KafkaUtils.generateRandomNameOfKafka;
 
 @Tag(REGRESSION)
+@SuiteDoc(
+    description = @Desc("Test suite for verifying Cluster Operator's ability to recover Kafka cluster components from various deletion and failure scenarios."),
+    beforeTestSteps = {
+        @Step(value = "Deploy shared Kafka cluster and KafkaBridge.", expected = "Kafka cluster with 3 replicas and KafkaBridge is deployed.")
```

Comment:
```suggestion
        @Step(value = "Deploy a Kafka cluster with node pools configured for 3 replicas and HTTP Bridge.", expected = "Kafka cluster with 3 replicas and HTTP Bridge is deployed.")
```

Reply:
Is Kafka Bridge required here.
Not mentioned in steps. If so, we should clarify the requirement
we're renaming Kafka Bridge :-)